app/components/UI/CryptoAmountInput.js

```diff
@@ -152,4 +124,4 @@ class WrappedCryptoAmountInputAsField extends React.Component {
   }
 }
 
-export default WrappedCryptoAmountInputAsField
+export default asField(withNumberInputMask(WrappedCryptoAmountInputAsField))
```

Comment:
Could haved used `compose` from `redux`, but that's just a matter of preference really